main.tex

```diff
@@ -1,6 +1,7 @@
 \documentclass[11pt]{article}
 \begin{document}
 
+    % Section A - one level of nesting
```

Comment:
Aren't there two levels of nesting? Or are you ignoring the implicit nesting of this section respective to the document root?

Reply:
Yeah I'm ignoring that.